lib/src/pylons_wallet/pylons_wallet_dev.dart

```diff
@@ -1,7 +1,114 @@
-import 'package:pylons_sdk/src/pylons_wallet/pylons_wallet_impl.dart';
+import 'dart:async';
+import 'package:pylons_sdk/src/features/models/sdk_ipc_message.dart';
+import 'package:pylons_sdk/src/generated/pylons/cookbook.pb.dart';
+import 'package:pylons_sdk/src/generated/pylons/item.pb.dart';
+import 'package:pylons_sdk/src/generated/pylons/payment_info.pb.dart';
+import 'package:pylons_sdk/src/generated/pylons/recipe.pb.dart';
+import 'package:pylons_sdk/src/features/models/sdk_ipc_response.dart';
+import 'package:uni_links_platform_interface/uni_links_platform_interface.dart';
+import '../../pylons_sdk.dart';
 
-/// Wallet used for testnet.
-/// TODO: does this even need to exist
```

Comment:
@afti-githobo what's your suggestion for this class, can you please elaborate?

Reply:
i think we should _probably_ eliminate the dev wallet class entirely unless there's a compelling reason not to, at this stage.

background on the dev wallet class:

at one point the idea was that we'd have a way to expose a consumer-level wallet that doesn't support recipe creation and a developer-level wallet that does, but ultimately a) it's unnecessary even in the case that a user isn't creating recipes b/c they can just... not create recipes b) minters and other "consumer" applications exist that actually do need to create recipes, so the assumption doesn't even hold.

at _present_ it's apparently a class for handling a wallet that works against the testnet (rather than the mainnet) but there's no implementation difference between writing functionality for one and the other, so the difference amounts to "it uses different urls and a different chain-id string." which is something you _could_ implement w/ object inheritance, ofc, but doesn't seem like a super elegant way to do it. so i think it'd prolly make more sense to just pass a singular pylons_wallet_imple class a configuration in the constructor. (you can then have static methods that create a wallet w/ the right config in the base class, so the usage would just be `PylonsWallet.mainnet()` or `PylonsWallet.testnet()`.

Reply:
agreed

Reply:
@afti-githobo Thanks for the great explanation. Completely agreed. Can you create a ticket to make it happen or do you need me to create one and assign it you? Let me know.

Reply:
i can do it, i just need to have my coffee before i get started in earnest

Reply:
@afti-githobo yes that was my reason to implement separate classes. And we can also do it with configurations. Thanks for pointing that out.